frontend/src/pages/AccountSent.vue

```diff
@@ -12,61 +12,81 @@
       </div>
     </div>
     <div v-else class="q-mx-auto" style="max-width: 800px">
-      <account-sent-table :sendMetadata="sendMetadata" />
+      <div v-if="needsSignature" class="text-center q-mb-md">
+        {{ $t('AccountSent.need-signature') }}
+        <base-button @click="getData" class="text-center q-mt-md" :label="$t('AccountSent.sign')" />
+      </div>
+      <div v-else-if="!needsSignature && sendMetadata.length > 0" class="q-mx-auto" style="max-width: 800px">
+        <account-sent-table :sendMetadata="sendMetadata" />
+      </div>
+      <div v-else-if="!needsSignature && sendMetadata.length === 0" class="q-mx-auto" style="max-width: 800px">
+        <account-sent-table :sendMetadata="sendMetadata" />
+      </div>
```

Comment:
Without this separate code path for the table we will sometimes see an empty table when it should be populated.

Reply:
I was able to remove this and add a loader in its place